chatbot/vector-db-persist-directory/resources/ml-fundamentals.txt

```diff
@@ -0,0 +1,30 @@
+#### ml-fundamentals:
```

Comment:
We should not hard core these file names. Instead, you should traverse the folder and dynamically generate this through a Python script. We can automate this later.

Reply:
In fact, all the links and the text like "#### ml-fundamentals:" are generated automatically by traversing the folder in Ocademy. So I think they are ok, and it is convenient to check the loaded data.

Reply:
If they are automatically generated, why you need to commit them? How could a new added MD be included then?